aptos-move/framework/aptos-framework/sources/aggregator/optional_aggregator.spec.move

```diff
@@ -69,48 +72,87 @@ spec aptos_framework::optional_aggregator {
 
     spec sub(optional_aggregator: &mut OptionalAggregator, value: u128) {
         include SubAbortsIf;
-        ensures ((optional_aggregator_value(optional_aggregator) == optional_aggregator_value(old(optional_aggregator)) - value));
+        ensures (
+            (
+                optional_aggregator_value(optional_aggregator) == optional_aggregator_value(
+                    old(optional_aggregator)
+                ) - value
+            )
+        );
```

Comment:
Specs seem to be weird with parenthesis and indentation

Reply:
Yes, my idea is that if there is only one set of parentheses nested within the parentheses, the indentation could offset the effect of two layers of parentheses to that of a single layer, and the inner set of parentheses would not be broken to a new line.